jepsen/src/jepsen/util.clj

```diff
@@ -188,6 +189,34 @@
              (recur))
          res#))))
 
+(defrecord Retry [bindings])
+
+(defmacro with-retry
+  "It's really fucking inconvenient not being able to recur from within (catch)
```

Comment:
On 02/15/2016 21:50, Tobias Schottdorf wrote:
> In jepsen/src/jepsen/util.clj
> <https://github.com/cockroachdb/jepsen/pull/2#discussion_r52942165>:

That's a change from upstream. :)